src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -355,6 +355,11 @@ case class ChannelSendError(info: Source.Verifier.Info) extends VerificationErro
   override def localMessage: String = s"The send expression ${info.trySrc[PSendStmt](" ")}might fail"
 }
 
+case class PredicateInstanceNoAccessError(info: Source.Verifier.Info) extends VerificationError {
+  override def localId: String = "predicate_instance_no_access_error"
+  override def localMessage: String = "Accessing predicate instance might fail"
```

Comment:
I would probably replace this with "Permission to predicate instance might not be available.", because it may be unclear what we mean by "accessing a predicate instance", but I leave this choice up to you.

Reply:
I was actually thinking about this as well but decided against it as this is also Viper's error message and the reason for this issue's test case specifies that the reason are insufficient permissions